src/ontology/mondo-edit.obo

```diff
@@ -520167,6 +520139,52 @@ is_a: MONDO:0003847 {source="https://orcid.org/0000-0001-5208-3432"} ! hereditar
 relationship: has_characteristic MONDO:0021152 {source="OMIMPS:616033"} ! inherited
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/6777" xsd:anyURI
 
+[Term]
+id: MONDO:0800451
+name: congenital amegakaryocytic thrombocytopenia
+subset: rare
+synonym: "congenital amegakaryocytic thrombocytopenia" EXACT CLINGEN_PREFERRED [OMIMPS:604498]
+xref: OMIMPS:604498 {source="MONDO:equivalentTo"}
+is_a: MONDO:0100241 {source="DOID:0090118", source="MESH:C535982", source="MONDO:Redundant"} ! inherited thrombocytopenia
+relationship: excluded_subClassOf MONDO:0001713 {source="Orphanet:3319", source="https://orcid.org/0000-0001-5208-3432"} ! inherited aplastic anemia
+relationship: has_characteristic MONDO:0021152 {source="OMIMPS:604498"} ! inherited
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/6622" xsd:anyURI
+property_value: seeAlso "https://rarediseases.info.nih.gov/diseases/640/congenital-amegakaryocytic-thrombocytopenia" xsd:anyURI {source="GARD:0000640"}
+
+[Term]
+id: MONDO:0800452
+name: congenital amegakaryocytic thrombocytopenia 1
+def: "A rare inherited bone marrow failure syndrome, in which the cause of the disease is a variation in the MPL gene. It is characterized by an isolated and severe decrease in the number of platelets and megakaryocytes during the first years of life that develops into bone marrow failure with pancytopenia later in childhood." [https://orcid.org/0000-0001-5208-3432, Orphanet:3319]
+subset: gard_rare {source="GARD:640"}
+subset: nord_rare {source="MONDO:NORD"}
+subset: ordo_disease {source="Orphanet:3319"}
+subset: orphanet_rare {source="Orphanet:3319"}
+subset: rare
+synonym: "amegakaryocytic thrombocytopenia, congenital" EXACT [MONDO:Lexical, OMIM:604498]
```

Comment:
This seems like it should be a synonym for MONDO:0800451 (congenital amegakaryocytic thrombocytopenia) based on the string value "amegakaryocytic thrombocytopenia, congenital". 

However, the value for [OMIM:604498]() is "AMEGAKARYOCYTIC THROMBOCYTOPENIA, CONGENITAL, 1; CAMT1" so the source information looks relevant for this term. Is the synonym value wrong?

Reply:
I updated this